### PR TITLE
feat(cmsis-pack): cmsis-pack for v8.3.0 release

### PR DESCRIPTION
**Pull request by GorgonMeducer**
### Description of the feature or fix

CMSIS-Pack for v8.3.0 release.

### Checkpoints
- [ ] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [ ] Update the documentation

### CODE REVIEW

**Comment by kisvegabor** [comment]
It might happen that a few commit will be added until tomorrow so I keep this PR open until tomorrow.

**Comment by GorgonMeducer** [comment]
@kisvegabor Then, before the final release, please let me know, so I can run the gen_pack.sh to update cmsis-pack

File: env_support/cmsis-pack/LVGL.lvgl.pdsc [hunk 1]
```diff
@@ -36,7 +36,13 @@
   <repository type="git">https://github.com/lvgl/lvgl.git</repository>
 
   <releases>
-    <release date="2022-06-29" version="1.0.5" url="https://raw.githubusercontent.com/lvgl/lvgl/master/env_support/cmsis-pack/LVGL.lvgl.1.0.5.pack">
+    <release date="2022-07-05" version="1.0.6" url="https://raw.githubusercontent.com/lvgl/lvgl/master/env_support/cmsis-pack/LVGL.lvgl.1.0.6.pack">
+      - LVGL 8.3.0
+      - Apply patch for memory leaking issue
+      - Apply patch to speed up non normal blend mode
+      - Other monior changes
```
Comment (embeddedt) [inline env_support/cmsis-pack/LVGL.lvgl.pdsc]:
```suggestion
      - Other minor changes
```

**Comment by kisvegabor** [comment]
@GorgonMeducer I started to work on the release. Please update the CMSIS-PACK one last time.

**Comment by kisvegabor** [comment]
Thank you!

**Comment by GorgonMeducer** [comment]
@kisvegabor You do it too quick... I haven't finished...

**Comment by kisvegabor** [comment]
Ahhh, sorry. Please open a new PR for release/v8.3 and I'll quickly fix it. Only the `tag` needs to be added again.

**Comment by GorgonMeducer** [comment]
https://github.com/lvgl/lvgl/pull/3453

**Pull request merged**